site/content/en/docs/installation/_index.md

```diff
@@ -151,9 +151,13 @@ data:
 __The `integrations.externalFrameworks` field is available in Kueue v0.7.0 and later.__
 
 > **Note**
-> See [Sequential Admission with Ready Pods](/docs/tasks/setup_sequential_admission) to learn
+> See [Sequential Admission with Ready Pods](/docs/tasks/manage/setup_sequential_admission) to learn
 more about using `waitForPodsReady` for Kueue.
 
+> **Note**
+> Setting `manageJobsWithoutQueueName` to `true` without disabling the `batch/job` integration
```

Comment:
This sounds too specific to the way OpenShift conducts operations.

Maybe you can prephrase like this:
Certain Kubernetes distributions might use batch/jobs to perform maintenance operations. For these distributions, setting ......

Reply:
Done.